modules/autofill/autofill-api/build.gradle

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.squareup.anvil'
+    id 'kotlin-android-extensions'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+    implementation Kotlin.stdlib.jdk7
+    implementation project(path: ':di')
+    implementation project(path: ':common')
+    implementation project(path: ':common-ui')
+
+    anvil project(path: ':anvil-compiler')
+    implementation project(path: ':anvil-annotations')
```

Comment:
I think we only need: 

    implementation KotlinX.coroutines.core
    implementation Google.android.material
    implementation AndroidX.appCompat
    implementation AndroidX.constraintLayout

Reply:
Done